datasets/ler/dummy/1.0.0/dummy_data/ler.conll

```diff
@@ -0,0 +1,8 @@
+Prozesskostenhilfe O
```

Comment:
Could you remove this file please ?

Reply:
Yes, removed it, thanks for the heads up.